utils/order-utils/src/orders.js

```diff
@@ -21,11 +21,14 @@ const signatures = require('./signatures')
 let nonce = 100
 
 const getLatestTimestamp = async () => {
-  return (await web3.eth.getBlock('latest')).timestamp
+  if (typeof web3 !== 'undefined') {
```

Comment:
can you add a quick comment on why web3 may be undefined?

Reply:
Not all environments have a global web3.